app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">SubscribersSearch</string>
+    <string name="title_activity_second">SecondActivity</string>
```

Comment:
No, I don'n have a second activity, these lines was a test because I created a class for a test